components/LoginForm.vue

```diff
@@ -71,7 +76,8 @@ export default Vue.extend({
     return {
       login: {
         username: '',
-        password: ''
+        password: '',
+        rememberMe: false
```

Comment:
Not essential, but I think that most people would expect this to be selected by default?

Reply:
I did it the same way current JF web does, but yeah I don't have a preference

Reply:
I stay with my opinion that, when we face an UI/UX design decision like this, we should look at what services that we and millions of people use daily (Spotify, Netflix, etc) do with the matter. They probably spent hours figuring this as well, so they know better than we do for sure.

Reply:
* Netflix: ticked
* Prime Video: not ticked
* Apple TV+: no choice available
* Disney+: I wasn't able to reach the password field as your email must first exist in their DB (gosh that's awful security practice)
* Spotify: ticked
* Deezer: no choice available
* Tidal: same as Disney+

It seems the vast majority of those having a tick I can see have them enabled by default.

Reply:
@Thibaultnocchi let's leave it ticked then! 😄

Reply:
It's ticked by default in the server, so it makes sense to have it ticked by default, I guess. If we make a change to that, it'd have to be project-wide, imo.

Reply:
Set to true!